src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -117,6 +117,14 @@ public function getName()
     }
 
     /**
+     * Disables use of stty.
```

Comment:
"Disables use ..." is probably not a good wording. Changed it to "Prevents usage of ...". It simply makes it optional to use stty which is currently mandatory when found on the system.